### ISSUE TITLE
Button randen - informatie/advies over randen van buttons toevoegen

### ISSUE THREAD

**Issue by RenateRoke** [nl-design-system/documentatie#2835]
## TLDR 

N.a.v. https://codefornl.slack.com/archives/C7KDQRXMZ/p1758635979376149?thread_ts=1758191217.034629&cid=C7KDQRXMZ willen we informatie toevoegen over randen en achtergrondkleuren van buttons en waarom het fijn is om ook daar aan contrast te denken.

## Acceptatiecriteria

- [ ] De button pagina bevat een best practice over randen/achtergrond met voldoende contrast.
